indexers/gemini-3h/accounts/src/mappings/helper.ts

```diff
@@ -1,53 +1,20 @@
-import { balance } from "@autonomys/auto-consensus";
-import { request } from "http";
+import { account } from "@autonomys/auto-consensus";
 import { createAndSaveBalanceHistory } from "./db";
-import { stringify } from "./utils";
 
 export const updateAccountBalance = async (
   accountId: string,
   blockNumber: bigint
 ) => {
-  const _balance = await balance(api as any, accountId);
+  const _account = await account(api as any, accountId);
```

Comment:
I've encountered with the same issue that leads to not be usable `account` without `as any`

Reply:
I'll add an issue